superset-frontend/src/addSlice/AddSliceContainer.tsx

```diff
@@ -88,69 +95,63 @@ export default class AddSliceContainer extends React.PureComponent<
 
   render() {
     return (
-      <div className="container">
-        <Panel>
-          <Panel.Heading>
-            <h3>{t('Create a new chart')}</h3>
-          </Panel.Heading>
-          <Panel.Body>
-            <div>
-              <p>{t('Choose a dataset')}</p>
-              <div style={styleSelectContainer}>
-                <Select
-                  clearable={false}
-                  ignoreAccents={false}
-                  name="select-datasource"
-                  onChange={this.changeDatasource}
-                  options={this.props.datasources}
-                  placeholder={t('Choose a dataset')}
-                  value={
-                    this.state.datasourceValue
-                      ? {
-                          value: this.state.datasourceValue,
-                        }
-                      : undefined
-                  }
-                  width={600}
-                />
-              </div>
-              <span className="text-muted">
-                {t(
-                  'If the dataset you are looking for is not available in the list, follow the instructions on how to add it in the Superset tutorial.',
-                )}{' '}
-                <a
-                  href="https://superset.apache.org/docs/creating-charts-dashboards/first-dashboard#adding-a-new-table"
-                  rel="noopener noreferrer"
-                  target="_blank"
-                >
-                  <i className="fa fa-external-link" />
-                </a>
-              </span>
-            </div>
-            <br />
-            <div>
-              <p>{t('Choose a visualization type')}</p>
-              <VizTypeControl
-                name="select-vis-type"
-                onChange={this.changeVisType}
-                value={this.state.visType}
-                labelType="primary"
-              />
-            </div>
-            <br />
-            <hr />
-            <Button
-              buttonStyle="primary"
-              disabled={this.isBtnDisabled()}
-              onClick={this.gotoSlice}
+      <StyledContainer className="container">
```

Comment:
I'm not sure if this className does anything... you might be able to get rid of it.